openjdk/excludes/ProblemList_openjdk21-openj9.txt

```diff
@@ -549,6 +550,7 @@ serviceability/jvmti/thread/GetStackTrace/getstacktr05/getstacktr05.java https:/
 serviceability/jvmti/vthread/VThreadNotifyFramePopTest/VThreadNotifyFramePopTest.java https://github.com/adoptium/aqa-tests/issues/1297 aix-all
 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#no-vmcontinuations https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
```

Comment:
`VThreadStackRefTest` already has an exclude for the `no-vmcontinuations` variant. You can remove `#no-vmcontinuations` to also disable the `default` variant of the test case.

Reply:
```suggestion
serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
```

Reply:
This comment is relevant for https://github.com/adoptium/aqa-tests/pull/4738.